src/pspm_emg_pp.m

```diff
@@ -1,29 +1,28 @@
 function [sts, outchannel] = pspm_emg_pp(fn, options)
 % ● Description
-%   pspm_emg_pp reduces noise in emg data in 3 steps. Following from the
-%   literature[1] it does the following steps:
-%   1)  Initial filtering:
-%       4th order Butterworth with 50 Hz and 470 Hz cutoff frequencies
-%   2)  Remove mains noise:
-%       50 Hz (variable) notch filter
-%   3)  Smoothing and rectifying:
+%   pspm_emg_pp pre-processes startle eyeblink EMG data in 3 steps, which
+%   were optimised in reference [1].
+%   (1)  Initial filtering:
+%       4th order Butterworth with 50 Hz and 470 Hz cutoff frequencies. 
+%   (2)  Removing mains noise:
+%       adjustable notch filter (default 50 Hz). 
+%   (3)  Smoothing and rectifying:
 %       4th order Butterworth low-pass filter with a time constant of 3 ms
-%       (=> cutoff of 53.05Hz)
-%   Once the data is preprocessed, according to the option 'channel_action',
-%   it will either replace the existing channel or add it as new channel to
-%   the provided file.
+%       (corresponding to a cutoff of 53.05 Hz). 
+%   While the input data must be an EMG channel, the output channel will be 
+%   of type emg_pp, as required by the startle eyeblink GLM.
 % ● Format
 %   [sts, channel_index]  = pspm_emg_pp(fn, options)
 % ● Arguments
-%                fn:  [string]
+% *              fn:  [string]
 %                     Path to the PsPM file which contains the EMG data.
-%           options:
-%       .mains_freq:  [integer] Frequency of mains noise to remove
-%                     with notch filter (default: 50Hz).
-%          .channel:  [numeric/string] Channel to be preprocessed.
-%                     Can be a channel ID or a channel name.
-%                     Default is 'emg' (i.e. last EMG channel)
-%   .channel_action:  ['add'/'replace'] Defines whether the new channel should
+%   ┌─────── options:
+%   ├────.mains_freq: [integer] Frequency of mains noise to remove
+%   │                 with notch filter (default: 50 Hz).
+%   ├───────.channel: [numeric/string] Channel to be preprocessed.
+%   │                 Can be a channel ID or a channel name.
+%   │                 Default is 'emg' (i.e. last EMG channel)
+%   └.channel_action: ['add'/'replace'] Defines whether the new channel should
 %                     be added or the previous outputs of this function should
 %                     be replaced. (Default: 'replace')
```

Comment:
There is an empty line above `History` section. Better to delete it to avoid potential mis-recognition.

Reply:
fixed.